main/models.py

```diff
@@ -263,28 +273,66 @@ def fees_to_cashflows(date, fees):
             for x in fees if x.date <= date]
 
 
-def revision_to_cashflows(rev, end_date):
+def revision_to_cashflows(rev, end_date, tenancy_begin_date = date(2015, 6, 1), tenancy_end_date = date(2015, 12, 13)):
```

Comment:
could you use the revision `start_date` and `end_date` instead of additionnal parameters ? `tenancy_begin_date` and `tenancy_end_date` are not used in the computations. I introduced them for historical reasons (tenants that were here before I created the system)